src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -425,6 +429,6 @@ public FormValidation doTestConnection(@QueryParameter URL serverUrl, @QueryPara
 
     @Override
     public String toString() {
-        return MoreObjects.toStringHelper(this).add("name", name).add("serverUrl", serverUrl).toString();
+        return "KubernetesCloud '" + name + "' serverUrl :" + serverUrl;
```

Comment:
My point is those are useful when they significantly make code less verbose, but can't see significant benefit here. Also always wonder about the actual performance impact for 5 method calls (even I know String concat is not a good sample about taking care for performances)